core/src/query/runner.rs

```diff
@@ -39,7 +36,7 @@ where
         let logger = self.logger.clone();
         let store = self.store.clone();
 
-        self.runtime.spawn(stream.for_each(move |query| {
+        tokio::spawn(stream.for_each(move |query| {
```

Comment:
Now that we are no longer passing the `reactor::Handle` are we creating more threads than before?

Reply:
I don't think we're creating more, but we're certainly using more threads. The `spawn` we were calling before apparently always spawned on the same thread as the event loop, which was the main thread.